daemon/cmd/endpoint.go

```diff
@@ -309,16 +309,14 @@ func (d *Daemon) createEndpoint(ctx context.Context, owner regeneration.Owner, e
 	if epTemplate.DatapathConfiguration == nil {
 		dpConfig := endpoint.NewDatapathConfiguration()
 		epTemplate.DatapathConfiguration = &dpConfig
-	}
-	if option.Config.EnableEndpointRoutes {
-		epTemplate.DatapathConfiguration.InstallEndpointRoute = true
-		epTemplate.DatapathConfiguration.RequireEgressProg = true
-		disabled := false
-		epTemplate.DatapathConfiguration.RequireRouting = &disabled
 	} else {
-		epTemplate.DatapathConfiguration.InstallEndpointRoute = false
-		epTemplate.DatapathConfiguration.RequireEgressProg = false
-		epTemplate.DatapathConfiguration.RequireRouting = nil
+		if option.Config.EnableEndpointRoutes {
+			epTemplate.DatapathConfiguration.InstallEndpointRoute = true
+			epTemplate.DatapathConfiguration.RequireEgressProg = true
+		} else {
+			epTemplate.DatapathConfiguration.InstallEndpointRoute = false
+			epTemplate.DatapathConfiguration.RequireEgressProg = false
+		}
```

Comment:
I did not move this block because of merge conflict this lines show as newly added lines , pls check with original checkout code.
I just removed refences to 'RequireRouting'.

Reply:
@qmonnet that was happened because of merge conflict, 
Thank you for finding and it was done now as you suggested.